conf/carbon.conf.example

```diff
@@ -311,6 +311,11 @@ MAX_DATAPOINTS_PER_MESSAGE = 500
 # thousands of TCP connections reduce the throughput of the service.
 #MAX_RECEIVER_CONNECTIONS = inf
 
+# Specify the user to drop privileges to
+# If this is blank carbon-relay runs as the user that invokes it
+# This user must have write access to the local data directory
```

Comment:
It was not intentional, I copied the entire paragraph initially. I've removed the lines you've mentioned and amended the change.